shared/base.js

```diff
@@ -29,15 +29,15 @@ function enableOrDisable(tabId, changeInfo, tab) {
   }
 }
 
-function reportIssue(tab, reporterID) {
+function reportIssue(tab, reporterID, tabId) {
```

Comment:
This looks good in terms of defining a function argument, but when `reportIssue` is getting called, there's no tab id getting passed in.

Reply:
@OrionStar25 thank you for your patience here!

So, I'm a little embarrassed to admit, but I lead you down the wrong path. Sorry for not paying closer attention. 🙈 

If you try this addon as-is, the screenshot uploading part is broken. It's because we passed in the `id` from the wrong tab, so we're executing the JS in the wrong place.

What we want to pass in is the tab ID of the *new* tab, not the tab the user clicked the addon in.

So you can remove this extra `tabId` argument here, and when `reportIssue` gets called.

Reply:
Thats alright! I'm trying to get this done : 

> If you try this addon as-is`

Pushed another commit with the changes though.